src/it/annotations-it/pom.xml

```diff
@@ -29,7 +29,6 @@
 
 
   <properties>
-    <sling.java.version>8</sling.java.version>
```

Comment:
Do we need to drop java 8 support for technical reasons? Otherwise, I would be inclined to keep it.

Reply:
Actually its working fine, let me keep the java 8 support.